### ISSUE TITLE
Update removed complete functionality

### ISSUE THREAD

**Issue by quassia** [BAndysc/WoWDatabaseEditor#91]
I've updated the app at the latest version (2021.1) and I am not able to use it:
Quick load -> Creature Script -> Pick any creature (e.g.: 3 Flesh Eater) -> notice that 'Click to Add Event' is disabled and not able to use it.
At right click the only option available is "Define global variable".

Platform: Windows 11
Core: TC WotLK

**Comment by BAndysc** [comment]
Are you sure nothing happens if you press "Add Event" button? This button is slightly grayed out and it doesn't have any background or border, but you can always click it. I've just tested in on a freshly downloaded version on Windows 11

**Comment by quassia** [comment]
Thank you for the reply. 
I've tried to click it, right-click it, click around it. Still does not work for me.

I've downgraded to an older version and it works: http://prntscr.com/1qdinoq  (installed this one back because I was afraid I configured the DB connection wrong somehow).

The new version looks like this: http://prntscr.com/1qdio1k
When I right-click it is what option I have: http://prntscr.com/1qdip4x
I can't even edit a previous scripted NPC (with another version of the app): http://prntscr.com/1qdipml
Maybe I am doing something wrong. 
Thanks for the help! Cheers :)

**Comment by BAndysc** [comment]
Oh, this "old version" is veeery old. Can you please tell me if the problem is still there if you switch appearance (File -> Settings) to Windows10 theme?

(so clicking Add Event button with left mouse button just does nothing, am I right?)

**Comment by gbmv33** [comment]
for me the problem its another if I click on anything on my computer with the program open the program blinks as if it had closed and opened very quickly and it won't let me do anything.
I have to keep extracting an old version to use and then it's impossible to use if I close the program because it updates automatically

**Comment by BAndysc** [comment]
@gbmv33 Hey, is there any chance you could record this behavior? I tried again to download [fresh version](https://github.com/BAndysc/WoWDatabaseEditor/releases/tag/2021.1) and again I didn't see any problem, do you start correct file (`WoWDatabaseEditorCore.Avalonia.exe` - the only application with icon)?  Is this the version you downloaded?

**Comment by gbmv33** [comment]
https://youtu.be/ahkTg8Pk1Zo

**Comment by quassia** [comment]
> Oh, this "old version" is veeery old. Can you please tell me if the problem is still there if you switch appearance (File -> Settings) to Windows10 theme?
> 
> (so clicking Add Event button with left mouse button just does nothing, am I right?)

Yes, it works if I change to "Windows10Dark" theme. Thank you! :) In my "MacOsCatalinaDark" theme clicking the grey text does not work. I'll stick with the Windows10Dark theme and report back if I encounter more bugs. 
Thank you again :)

**Comment by BAndysc** [comment]
@quassia 
> Yes, it works if I change to "Windows10Dark" theme. Thank you! :) In my "MacOsCatalinaDark" theme clicking the grey text does not work. I'll stick with the Windows10Dark theme and report back if I encounter more bugs.
> Thank you again :)

Thanks for the bug report, indeed, I was able to reproduce the bug with MacOS theme, I've fixed that so if you like this one, you can use this one again :)

@gbmv33
Hmm, so this is a completely different bug, looks weird. I have updated GUI library to the latest version, maybe it will fix the bug? Can you please try to upgrade the editor and test?

**Comment by gbmv33** [comment]
works great now :D

**Comment by gbmv33** [comment]
I was wrong, after 10 seconds with the program open, the bug returned

**Comment by quassia** [comment]
> Thanks for the bug report, indeed, I was able to reproduce the bug with MacOS theme, I've fixed that so if you like this one, you can use this one again :)
Many thanks. I can confirm it works again.
/bow
